__tests__/app.test.js

```diff
@@ -29,34 +29,116 @@ describe("GET /api/topics", () => {
         body.topics.forEach((topic) => {
           expect(topic).toMatchObject({
             slug: expect.any(String),
-            description: expect.any(String)
+            description: expect.any(String),
           });
         });
       });
   });
 });
 
 describe("GET /api/articles", () => {
-    it("200: responds with an array of article objects, each containing author, title, article id, topic, creation time, votes, and article image url properties. Each object should also have a comment count property, calculated using queries. The articles should be ordered by date in descending order", () => {
-      return request(app)
-        .get("/api/articles")
-        .expect(200)
-        .then(({ body }) => {
-          expect(body.articles).toBeInstanceOf(Array);
-          expect(body.articles).toBeSorted({key: 'created_at', descending: true});
-          expect(body.articles.length).toBeGreaterThan(0);
-          body.articles.forEach((article) => {
-            expect(article).toMatchObject({
-                article_id: expect.any(Number),
-                title: expect.any(String),
-                topic: expect.any(String),
-                author: expect.any(String),
-                created_at: expect.any(String),
-                votes: expect.any(Number),
-                article_img_url: expect.any(String),
-                comment_count: expect.any(Number)
-            });
+  it("200: responds with an array of article objects, each containing author, title, article id, topic, creation time, votes, and article image url properties. Each object should also have a comment count property, calculated using queries. The articles should be ordered by date in descending order", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles).toBeInstanceOf(Array);
+        expect(body.articles).toBeSorted({
+          key: "created_at",
+          descending: true,
+        });
+        expect(body.articles.length).toBeGreaterThan(0);
+        body.articles.forEach((article) => {
+          expect(article).toMatchObject({
+            article_id: expect.any(Number),
+            title: expect.any(String),
+            topic: expect.any(String),
+            author: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+            comment_count: expect.any(Number),
           });
         });
+      });
+  });
+});
+
+describe.only("GET /api/articles/:id", () => {
```

Comment:
please make sure that you dont push any work that has a .only on, as knowing that your tests are passing is our only way we know all of your code is working (normally I would reject the PR right away) but this time I'll warn you if I see then in later I'll just reject your pr :)